### DISCUSSION TITLE
printing head stops momentarily for about 1 second after every retraction

### DISCUSSION THREAD

**Discussion by manpres2** [Jyers/Marlin#1026] [Q&A] [answered]
Hi,

This is my first post on github so please let me know if i missed something while asking for help.
 i have Ender 3 V2 on creality 4.2.2 board. I installed 1.3.4 HS 5x5 and 3x3 both the firmware are giving me the same issue. 

issue: print head stop momentarily during retraction for 1 second or less. i was printing a wheel with spokes and when ever it retracts for printing spokes it will halt for approx a second. I am back to 1.3.3 now and everything is back to normal.

**Comment by Cal-LF** [comment]
Try resetting the eprom after each firmware install.

**Reply by Manolets96** [reply]
I have the same problem, I did what I said and the problem continues, stops 1 sec or more and filament keeps coming out causes damage to print

**Reply by Manolets96** [reply]
Thanks for help ;D

**Comment by Jyers** [comment, marked as answer]
Another thing to try would be to disable powerloss recovery in the advanced menu. If you have a slow SD card it can sometimes take a bit for the file to save, causing the printer to pause for a moment.

**Reply by Manolets96** [reply]
The solution to the problem is this, the SD card is slow, I disabled this option and everything is working correctly. Good work, Thanks ;D

**Reply by manpres2** [reply]
my card was not slow for the previous firmware, what could have caused this to be slow now. oh and i am using Octoprint.

**Reply by Jyers** [reply]
SD cards do get slower over time as they wear, especially if it's the cheap one that came with the machine

**Reply by manpres2** [reply]
this issue is happening with the current version only and not with the previous 1.3.3. I think it's not due to memory card as i have class 10 memory card for both machine and octoprint.

**Reply by manpres2** [reply]
> 
> 
> this issue is happening with the current version only and not with the previous 1.3.3. I think it's not due to memory card as i have class 10 memory card for both machine and octoprint.

Hi,

I changed the memory card to samsung class 10 and it is still happening. i tried disabling the power loss recovery and its still same.

version 1.3.3 works fine so i am still on that version.

**Comment by Tuncay-Ayhan** [comment]
What device is your Octoprint instance running on?

Op wo 23 jun. 2021 om 09:35 schreef manpres2 ***@***.***>

**Reply by manpres2** [reply]
i am running on raspberry pi 3B

**Reply by smooreace** [reply]
That certainly isn't helping things!

**Comment by Tuncay-Ayhan** [comment]
Mine had similar issues. It stopped in the middle of a print (yesterday).
3B+. What a coincidence.

I just got a Chromebox CXI2, installed Ubuntu on it. Working buttery smooth
😁. Load on CPU is max 7%. Also RAM usage is 17%.

The only thing it’s limiting, the baud rate. I can live with that.



Op wo 23 jun. 2021 om 17:01 schreef smooreace ***@***.***>

**Reply by manpres2** [reply]
this is issue is only happening with this version and not happening with 1.3.3. i have reverted back to 1.3.3. Can any one please  point me to 1.3.3 High speed firmware.

**Reply by smooreace** [reply]
Huh?

**Reply by manpres2** [reply]
> 
> 
> Huh?

can you please elaborate what you meant by that "Huh?" my request was simple if some one can point me to the previous version HS version and that implies i have the 9 point slow probe version

**Reply by Jyers** [reply]
https://github.com/Jyers/Marlin/releases

**Reply by manpres2** [reply]
> 
> 
> @ manpres2
> I'm sorry for what happened to you but I have to say that the firmware is not involved. Understand that we would already be assaulting Jacob for a long time if that was the case. I installed and configured a dozen printers with the base firmware 1.3.4 and I never had your situation. On the other hand, I always do a reset of the printers with each update to purge the EEprom. Note that the more features there will be that require saving in the EEprom, it will always have to be done or otherwise random parameters will be assigned to others which may cause oddities. And don't forget that an sdcard can always be faulty and have errors during writes and cause slowdowns, also faulty for example if it is badly formatted without the right parameters.
> Finally there is indeed a change but in the base code of Marlin on the Power Loss Recovery which since the last Marlin bugfix writes more often on the SD ...

Hi,  I checked all the replies and tried all solutions . I have changed the memory card to class 10 samsung, issue is still same. 
This does not happen with version 1.3.3.

I tried disabling the power failure feature as well but some reason it only worked once
